### PR TITLE
bsdlib: Update BSD library for nrf9160.

### PR DESCRIPTION
**Pull request by gbakke**
Library version a9c184efc8c.

Signed-off-by: Glenn Ruben Bakke <glenn.ruben.bakke@nordicsemi.no>

### CODE REVIEW

**Comment by gbakke** [comment]
CC @rlubos , @ru-fu , @oxalistriangularis

File: bsdlib/CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,13 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.2.2
+************
+
+Updated library with API for setting APN name when doing getaddrinfo request.
+
+* Providing API through nrf_getaddrinfo, ai_next to set a second hint which defines APN name to use for getaddrinfo query. The hint must be using NRF_AF_LTE, NRF_SOCK_MGMT, NRF_PROTO_PDN as family, type and protocol. The APN is set through the ai_canonname field.
```
Comment (ru-fu) [inline bsdlib/CHANGELOG.rst]:
```suggestion
* Providing API through nrf_getaddrinfo, ai_next to set a second hint that defines the APN name to use for getaddrinfo query. 
 The hint must be using NRF_AF_LTE, NRF_SOCK_MGMT, and NRF_PROTO_PDN as family, type, and protocol.
 The APN is set through the ai_canonname field.
```

**Comment by gbakke** [comment]
merged doc change proposal and rebased.

**Comment by oxalistriangularis** [comment]
I am assuming there are no header changes even though library has changed. The change log is misleading as this commits introduces new socket types with no changes in the header!

I approve on the condition that header changes were not needed for this revision.

**Pull request merged**